factorial/fact.cpp

```diff
@@ -0,0 +1,36 @@
+#include<iostream>
+int fun(int n)
+{
+if(n==0)
```

Comment:
Is this case only applicable for n is equal to 0 or should it also be the case for n == 1

Reply:
for n==1 function will give correct output, please review